src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+
+import style from './Dashboard.module.scss';
+
+const activities = [
+  { activity: 'running', id: 1, timesDone: 3 },
+  { activity: 'swimming', id: 2, timesDone: 2 },
+  { activity: 'not smoking', id: 3, timesDone: 21 },
+];
+
+export const Dashboard = () => {
+  const [formVisible, setFormVisible] = useState<boolean>(false);
+  const [newActivity, setNewActivity] = useState<string>('');
```

Comment:
Good job using the explicit types! It is great to see it didn't scare you. 

Just as a measure of making stuff easier. We don't need to use explicit types, since we have default values in the states typescript will infer them (guess them) implicitly

So this is the same
```suggestion
  const [formVisible, setFormVisible] = useState(false);
  const [newActivity, setNewActivity] = useState('');
```